src/Tribe/Service_Providers/Context.php

```diff
@@ -259,21 +259,39 @@ static function ( $data ) {
 				],
 				'tec_post_type'        => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								$found = [
+									! empty( tribe_get_request_var( TEC::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Organizer::POSTTYPE, false ) ),
+								];
+								$found = array_filter( $found );
+
+								return count( $found ) ?: Tribe__Context::NOT_FOUND;
```

Comment:
:no_entry_sign: Using short ternaries is not allowed

<sub>hash: b4de26a2a672e063cea2fc6ef2cb691d</sub>